lib/src/scan.dart

```diff
@@ -58,11 +58,11 @@ List<T> parseList<T>(StringScanner scanner, T Function() parseElement) {
 ///
 /// If [name] is passed, it's used to describe the expected value if it's not
 /// found.
-String expectQuotedString(StringScanner scanner, {String name}) {
+String expectQuotedString(StringScanner scanner, {String? name}) {
```

Comment:
@natebosch – thoughts here about populating the default value of `name` here?

It's a kinda breaking change – but unlikely to affect anyone.